person.rb

```diff
@@ -1,7 +1,7 @@
 require_relative 'nameable'
```

Comment:
- [ ] kindly remove the nameable as there is no class here with it in this milestone

Reply:
But if I remove require_relative I think there will be rubocop errors.